### PR TITLE
Reveal syntax error in parsing .mention-bot file by leaving a comment

### PR DESCRIPTION
**Pull request by jeffreyleeon**
Refs #188

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -163,6 +163,16 @@ async function work(body) {
       headers: {
         Accept: 'application/vnd.github.v3.raw+json'
       }
+    }).catch(function(e) {
+      if (e instanceof SyntaxError && repoConfig.actions.indexOf(data.action) !== -1) {
+        // Syntax error while reading custom configuration file
+        var message =
+          'Unable to parse mention-bot custom configuration file due to syntax error.\n' +
```
Comment (vjeux) [inline server.js]: due to `a` syntax error

File: server.js [hunk 2]
```diff
@@ -163,6 +163,16 @@ async function work(body) {
...
+          'Please check the potential errors below:\n\n' +
```
Comment (vjeux) [inline server.js]: Potential `root causes` below

File: server.js [hunk 3]
```diff
@@ -163,6 +163,16 @@ async function work(body) {
...
+          '1. Invalid JSON type\n' +
```
Comment (vjeux) [inline server.js]: 1. Having comments

**Comment by jeffreyleeon** [comment]
@vjeux Ok, please check if it looks better :)

**Comment by vjeux** [comment]
Yay, perfect thanks :)

I'll push it to the live instance this evening

**Pull request merged**